ext/standard/assert.c

```diff
@@ -62,12 +72,54 @@ static PHP_INI_MH(OnChangeCallback) /* {{{ */
 }
 /* }}} */
 
+static PHP_INI_MH(OnUpdateActiveBool)
+{
+	bool *p = (bool *) ZEND_INI_GET_ADDR();
+	*p = zend_ini_parse_bool(new_value);
+	if ( (stage != ZEND_INI_STAGE_DEACTIVATE && stage != ZEND_INI_STAGE_SHUTDOWN && stage != ZEND_INI_STAGE_ASSERT_OPTIONS) && !*p) {
+		php_error_docref(NULL, E_DEPRECATED, "assert.active INI setting is deprecated");
+	}
+	return SUCCESS;
```

Comment:
might be nicer to introduce some inline function for this

Reply:
Done!